packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -861,7 +861,7 @@ class SelectionOverlay {
         context: context,
         below: magnifierConfiguration.shouldDisplayHandlesInMagnifier
             ? null
-            : _handles!.first,
```

Comment:
Should the `|| _handles == null` above on line 856 be removed?

Reply:
Yes thank you!  I think you added that check just before I created this PR and I somehow totally missed it.  Likely your PR has already fixed the crash that I was trying to fix with this PR.  I think this PR is still worthwhile to allow the magnifier to show without the handles being visible though.  Let me know what you think of the code now after my change.

Reply:
Yeah I added that check, and I agree this is still a worthwhile change.